e107_core/controllers/system/xup.php

```diff
@@ -14,6 +14,7 @@
  *    $Author: e107coders $
 */
 
+e107::lan('social', false, true);
 

```

Comment:
You're loading a plugin language file for a system script. `e107_core/controllers/system/xup.php` 
**lan_user.php** would be more appropriate. (see line 120+)